### PR TITLE
Fix navbar

### PR DESCRIPTION
**Pull request by Safisyx**
So this is an idea for making the navBar staying after we reload the page.
I added a new route in the backend, will do a pull request for that right after this.

But as a remark, it's weird to see the navBar re-render :smiley: 
An idea on how to fix that is to not render anything or maybe just render the top bar without drawer before getting the state updated. But we can discuss about that tomorrow.

Also, I started this pull request from the Nigel branch to avoid any conflicts later

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -140,6 +139,7 @@ class NavBar extends PureComponent {
           }}
           onLeftIconButtonClick={_=> this.setState({drawer: !this.state.drawer})}
         />
+        {
```
[the remark this answers is not in the document]
Comment (Safisyx) [reply src/components/NavBar.js]: I put some code before and I just forgot to remove them 😄

**Comment by Safisyx** [comment]
That's because it's waiting for the response in the back end haha. We can put something like 'loading...' maybe 😛

File: src/components/NavBar.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ju1111) [reply src/components/NavBar.js]: ahhhh haha I see XD

**Comment by Ju1111** [comment]
but the user is actually logged out when reloading, right?

**Comment by Safisyx** [comment]
No the user is not logged out when reloading since we are storing the jwt

**Comment by Ju1111** [comment]
should we fix that then and delete the jwt when logging out? then we don't have the problem with the navbar anymore, right?

**Pull request merged**